src/city/health.c

```diff
@@ -201,6 +203,32 @@ static void cause_disease(int total_people)
     }
 }
 
+static void reduce_sickness_level_in_building(building *b)
+{
+    if (!b->has_plague) {
+            if (b->sickness_level) {
+                int city_global_coverage = city_health(); // reduction
+                int city_hospital_coverage = 100 - city_culture_coverage_hospital(); // malus
```

Comment:
I think the variable name could be renamed, from what I get it's percentage of people without coverage.

Reply:
ok